### PR TITLE
Exported and added typings to options and verify function

### PR DESCRIPTION
**Pull request by dwilt**
See here for more info: https://github.com/andyrichardson/passport-workos/issues/2

### CODE REVIEW

**Comment by andyrichardson** [comment]
I'm happy to merge this - mind rebasing against master so I can be sure tests are working :pray:

**Comment by dwilt** [comment]
Will do this tonight. OOO for today

**Comment by dwilt** [comment]
@andyrichardson says I'm up to date?

```
dwilt@Dans-MBP passport-workos % git rebase master
Current branch Add-typings-to-options-and-verify-function is up to date.
dwilt@Dans-MBP passport-workos % 
```

File: package.json [hunk 1]
```diff
@@ -12,7 +12,6 @@
     "passport",
     "workos"
   ],
-  "main": "dist/index.js",
```
Comment (dwilt) [inline package.json:15]: There was an extra `main` prop in `package.json`

**Comment by andyrichardson** [comment]
> @andyrichardson says I'm up to date?
> 
> ```
> dwilt@Dans-MBP passport-workos % git rebase master
> Current branch Add-typings-to-options-and-verify-function is up to date.
> dwilt@Dans-MBP passport-workos % 
> ```

That'll likely be because you're rebasing on the master branch of your fork. Off the top of my head, you'll need to run something like this

```sh
git remote add-url upstream https://github.com/andyrichardson/passport-workos.git
git pull --rebase upstream master
```

**Comment by dwilt** [comment]
@andyrichardson ahhh, yep, got it, thanks. Learning a lot! Ran them locally (I'm assuming your CI will run them too though)

```
 PASS  src/index.test.ts
  on login
    on no query params
      ✓ passport.js throws unauthorized error (27 ms)
    on connection
      ✓ calls workos api with connection (6 ms)
      ✓ redirects to login url (5 ms)
    on domain
      ✓ calls workos api with domain (3 ms)
      ✓ redirects to login url (3 ms)
    on email
      ✓ calls workos api with domain (3 ms)
      ✓ redirects to login url (2 ms)
  on callback
    ✓ calls getProfileAndToken with code and clientID (3 ms)
    on invalid authorization code
      ✓ passport.js throws unauthorized error (3 ms)
    on successful code verification
      ✓ calls verify function (4 ms)
    on user verify fn success
      ✓ forwards to next handler (4 ms)
    on user verify fn error
      ✓ throws error (4 ms)
    on user verify fn not authorized
      ✓ passport.js throws unauthorized error (2 ms)

Test Suites: 1 passed, 1 total
Tests:       13 passed, 13 total
Snapshots:   3 passed, 3 total
Time:        4.701 s
Ran all test suites.
dwilt@Dans-MBP passport-workos % 
```

**Comment by andyrichardson** [comment]
Yup looks good to me! 

Can you do a quick run of prettier on the file `$(npm bin)/prettier --write src/index.ts` and then I'll merge this in :+1:

**Comment by dwilt** [comment]
done!

**Pull request merged**